lib/hammer_cli_katello/ping.rb

```diff
@@ -44,7 +44,8 @@ class PingCommand < HammerCLIKatello::Command
     def execute
       d = send_request
       print_data d
-      d['status'] != _("FAIL") ? HammerCLI::EX_OK : 1
+      service_failure = d['services'].values.map {|v| v['status'] }.include?(_("FAIL"))
```

Comment:
Could you please elaborate? I'm having trouble seeing how `any?` would work in this situation.